python/session-02.md

```diff
@@ -0,0 +1,111 @@
+# SET
```

Comment:
## Python Session-02
### Chapter 2 of Python Distilled: Operators, Expressions, and Data Manipulation
- # SET